front/lib/plans/subscription.ts

```diff
@@ -463,3 +465,32 @@ export const updateWorkspacePerMonthlyActiveUsersSubscriptionUsage = async ({
     }
   }
 };
+
+/**
+ * Proactively cancel inactive trials.
+ */
+export async function maybeCancelInactiveTrials(
```

Comment:
NIT suggestion: Since this will I believe not be reusable anywhere else I would kill and do that directly on the webhook. ff we want to keep here I would just rename to be clearer -> `cancelSubscriptionIfInactiveTrial`

Reply:
I plan to use it in a script to cancel the trials that won't be captured by this change.